File: INICIANTE/1101 Sequência de Números e Soma.py
def somaDaSequencia(num1, num2):
	maior = menor = soma = 0
	if num1 > num2:
		maior = num1
		menor = num2
	else:
		maior = num2
		menor = num1
	for i in range(menor, maior+1):
		print(i, end=' ')
		soma += i
	msg = 'Sum=%d' % soma
	return msg

valor1, valor2 = map(int, input().split())
while valor1 > 0 and valor2 > 0:
	somador = somaDaSequencia(valor1, valor2)
	print(somador)
	valor1, valor2 = map(int, input().split())

File: C/INICIANTE/1101 Sequência de Números e Soma.c
#include <stdio.h>

int main()
{
    int v1, v2, menor, maior, cont, soma=0;
    scanf("%d %d", &v1, &v2);
    while (v1 > 0 && v2 > 0)
    {
        if (v1 > v2)
        {
            maior = v1;
            menor = v2;
        }
        else
        {
            maior = v2;
            menor = v1;
        }
        cont = menor;
        while (cont != (maior+1))
        {
            printf("%d ", cont);
            soma += cont;
            cont += 1;
        }
        printf("Sum=%d\n", soma);
        soma = 0;
        scanf("%d %d", &v1, &v2);
    }
    return 0;
}
